### PR TITLE
add tier2 test for cockpit webconsole integration

### PR DESCRIPTION
**Pull request by tstrych**
Changing PR: #7554
Moving test more into airgun.
Airgun PR [466](https://github.com/SatelliteQE/airgun/pull/466)

test results:
```
test_host.py::test_positive_cockpit 
=================== 1 passed, 29 deselected in 95.11 seconds ===================
```

### CODE REVIEW

**Comment by jameerpathan111** [comment]
@pondrejk we should close PR #7554, now that we have this one.

File: tests/foreman/ui/test_host.py [hunk 1]
```diff
@@ -1908,3 +1908,27 @@ def test_positive_gce_cloudinit_provision_end_to_end(
...
+    :id: 5a9be063-cdc4-43ce-91b9-7608fbebf8bb
+
+    :expectedresults: Cockpit page is loaded and displays sat host info
+
+    :CaseLevel: System
+
+    """
+    with session:
+        session.organization.select(org_name='Default Organization')
+        session.location.select(loc_name='Any Location')
```
Comment (JacobCallahan) [inline tests/foreman/ui/test_host.py]: does this only work with the default org and any location selected?
if not, was there a specific need to select them here?

Comment (mirekdlugosz) [reply tests/foreman/ui/test_host.py]: This needs host, and uses Satellite to save time. Satellite is assigned to default organization and default location and won't be displayed in table when any other combination is selected.

Comment (JacobCallahan) [reply tests/foreman/ui/test_host.py]: interesting. thanks for the explanation

**Pull request merged**